templates/skeleton/app/routes/account.$.tsx

```diff
@@ -2,13 +2,11 @@ import {redirect, type LoaderFunctionArgs} from '@shopify/remix-oxygen';
 
 // fallback wild card for all unauthenticated routes in account section
 export async function loader({context}: LoaderFunctionArgs) {
-  if (await context.customerAccount.isLoggedIn()) {
-    return redirect('/account', {
-      headers: {
-        'Set-Cookie': await context.session.commit(),
-      },
-    });
-  }
+  await context.customerAccount.handleUnauthorized();
```

Comment:
Is there a way to share this logic in a layout? e.g a pathless layout that just checks for authorization?

Reply:
Is that not `templates/skeleton/app/routes/account.tsx` ?